.circleci/wait.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+wait_for_cassandra () {
+  for i in 1 2 3 4 5; do
+    if docker exec $1 nodetool status | grep "^UN"; then
```

Comment:
Double quote to prevent globbing:

```suggestion
    if docker exec "$1" nodetool status | grep "^UN"; then
```